<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditAdminWindow</class>
 <widget class="QWidget" name="AuditAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{border-radius: 10px;}
border: none;</string>
  </property>
  <widget class="QFrame" name="BackGroundFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 0;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="TopFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 169, 239);
border: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="LogoFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>300</width>
       <height>120</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="TitleLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>25</y>
        <width>131</width>
        <height>36</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>YouSheBiaoTiHei</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);
font-size:32px;
</string>
      </property>
      <property name="text">
       <string>国联易安</string>
      </property>
     </widget>
     <widget class="QLabel" name="VersionLabel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>35</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.505682, stop:0 rgba(250, 193, 31, 255), stop:1 rgba(252, 228, 4, 255));
color: rgb(156, 69, 21);
border-radius: 10;
font-size:13px;</string>
      </property>
      <property name="text">
       <string>  (主机版) V2.0 </string>
      </property>
     </widget>
     <widget class="QLabel" name="SloganLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>YouSheBiaoTiHei</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size:19px;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>恶意代码辅助监测系统</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="ButtonFrame">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>360</width>
       <height>120</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="UserButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: qlineargradient(spread:pad, x1:0.985075, y1:1, x2:1, y2:0, stop:0 rgba(118, 180, 242, 255), stop:1 rgba(167, 203, 240, 255));
}
:hover{
	background-color: qlineargradient(spread:pad, x1:0.985075, y1:1, x2:1, y2:0, stop:0 rgba(118, 180, 242, 255), stop:1 rgba(167, 203, 240, 255));
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../statics/imgs/user.png</normaloff>../statics/imgs/user.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="UserButtonName">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>88</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;
font-size:13px;</string>
      </property>
      <property name="text">
       <string>日志审计</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="MenuFrame">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>0</y>
       <width>240</width>
       <height>120</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="CloseButton">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>5</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">:hover{
	background-color: qlineargradient(spread:pad, x1:0.985075, y1:1, x2:1, y2:0, stop:0 rgba(118, 180, 242, 255), stop:1 rgba(167, 203, 240, 255));
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../statics/imgs/close.png</normaloff>../statics/imgs/close.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>23</width>
        <height>23</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="MiniButtuon">
      <property name="geometry">
       <rect>
        <x>155</x>
        <y>5</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">:hover{
	background-color: qlineargradient(spread:pad, x1:0.985075, y1:1, x2:1, y2:0, stop:0 rgba(118, 180, 242, 255), stop:1 rgba(167, 203, 240, 255));
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../statics/imgs/mini.png</normaloff>../statics/imgs/mini.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>18</width>
        <height>18</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="SetButtuon">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>5</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">:hover{
	background-color: qlineargradient(spread:pad, x1:0.985075, y1:1, x2:1, y2:0, stop:0 rgba(118, 180, 242, 255), stop:1 rgba(167, 203, 240, 255));
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../statics/imgs/more.png</normaloff>../statics/imgs/more.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="SkinButton">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>5</y>
        <width>30</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">:hover{
	background-color: qlineargradient(spread:pad, x1:0.985075, y1:1, x2:1, y2:0, stop:0 rgba(118, 180, 242, 255), stop:1 rgba(167, 203, 240, 255));
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../statics/imgs/skin.png</normaloff>../statics/imgs/skin.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="LogoutButton">
      <property name="geometry">
       <rect>
        <x>165</x>
        <y>70</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	color: red;
	border-radius: 15;
}
:hover{
	background-color: rgb(145, 193, 243);
}</string>
      </property>
      <property name="text">
       <string>登出</string>
      </property>
     </widget>
     <widget class="QLabel" name="UserTypeLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>71</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>安全审计员</string>
      </property>
     </widget>
     <widget class="QLabel" name="AvatarLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>75</y>
        <width>2</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QFrame" name="MoreMenuFrame">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>35</y>
        <width>111</width>
        <height>60</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(154, 209, 252);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="ChangePassButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>30</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">:hover{
 color: black;
}</string>
       </property>
       <property name="text">
        <string>修改密码</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AboutButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>111</width>
         <height>30</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">:hover{
 color: black;
}</string>
       </property>
       <property name="text">
        <string>关于</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="BottomFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>900</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0, cy:1, radius:1.02262, fx:0, fy:1, stop:0 rgba(60, 86, 222, 255), stop:0.995025 rgba(138, 206, 253, 250));
border: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>460</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:10px;</string>
     </property>
     <property name="text">
      <string>产品授权:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>460</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:10px;</string>
     </property>
     <property name="text">
      <string>产品版本:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>460</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:10px;</string>
     </property>
     <property name="text">
      <string>2.0.0.05041735</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>460</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
font-size:10px;</string>
     </property>
     <property name="text">
      <string>未授权</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="LogFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>900</width>
      <height>460</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qradialgradient(spread:pad, cx:0, cy:1, radius:1.02262, fx:0, fy:1, stop:0 rgba(60, 86, 222, 255), stop:0.995025 rgba(138, 206, 253, 250));
border: none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="ToolsBackFrame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>900</width>
       <height>460</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="XitongFuzhuFrame_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>65</y>
        <width>821</width>
        <height>381</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <underline>true</underline>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(146, 178, 242, 80);
border-radius: 20;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTableWidget" name="LogTableWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>821</width>
         <height>381</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:transparent;</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_38">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font-size:25px;</string>
      </property>
      <property name="text">
       <string>日志审计</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>BottomFrame</zorder>
   <zorder>TopFrame</zorder>
   <zorder>LogFrame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
